core/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-add_library(prometheus-cpp-core
+
+add_library(core
```

Comment:
The problem with renaming the targets from `prometheus-cpp-core` to `core` is that also the library files get the short name:
```
./core/libcore.a
./pull/libpull.a
./push/libpush.a
```
I'd prefer if you restore the long names and set the `EXPORT_NAME`, again.

Thanks,
Gregor

Reply:
Hi. Would you be ok to make your request more explicit, i.e., explicitly stating to give the library a specific name:

https://github.com/jupp0r/prometheus-cpp/pull/166/files#diff-5f4a237561b8048385b3ba09ef74fc96R30

while naming the targets internally and externally the same.